css/style.css

```diff
@@ -378,7 +378,7 @@ form {
   text-align: center;
   align-items: center;
   gap: 20px;
-  background-image: url(/images/formbackgroungimg.png);
+  background-image: url(/images/illustration-contact-form.svg);
```

Comment:
- [ ] As requested by the previous reviewer https://github.com/stephenkati/Portfolio-setup-and-mobile-first/pull/5#discussion_r1073146203 Please fix the path to you image assets correctly so it is displayed on all devices just as mentioned by the previous reviewer, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213117328-56acb4a3-7377-4c49-b1dc-fdcaaa39e35c.png)

notice how the image for medium icon is broken and also the background image is not showing up, as recommended earlier  correct path for your medium icon = `<img src="./images/iconmonstr-medium-1.svg">`
correct path for you background image =` background-image: URL(../images/illustration-contact-form.svg);
`